tests/sentry/tsdb/test_redis.py

```diff
@@ -386,6 +402,22 @@ def test_frequency_tables(self):
             },
         }
 
+        self.db.delete_frequencies(
```

Comment:
could be nice to have a test case where we delete just a sub-time-interval of the data and assert on the stuff that remains? current cases all just gut the dataset entirely unless I'm misreading something